RunoPW/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "author": {
+    "name": "め Kerem",
+    "id": "665482610859966464"
+  },
+  "contributors": [{
+    "name": "Runo",
+    "id": "702935358395908168"
+  }],
+  "service": "Runo",
+  "description": {
+    "en": "Make Friends & Chat With Millions In A Virtual World",
+    "tr": "Müthiş İnsanlarla Dolu İlginç Bir Yer"
+  },
+  "url": "runo.pw",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/utrvCGI.png",
+  "thumbnail": "https://i.imgur.com/utrvCGI.png",

```

Comment:
you sitll need to do a thumbnail

Reply:
I prefer it

Reply:
I enjoy

Reply:
You are not allowed to do it like this, so change it or your PR will never be accepted.